### DISCUSSION TITLE
Kayles Oyunu | SonsuzUs Github

### DISCUSSION THREAD

**Discussion by giscus** [sonsuzus/sonsuzus.github.io#127] [Site]
# Kayles Oyunu | SonsuzUs Github

Bu oyun iki kişi arasında ve yanyana dizili 13 taşla oynanıyor. İki oyuncu da sırayla aşağıdaki hamlelerden birini yapıyor:

https://sonsuzus.github.io/posts/kayles-oyunu/

**Comment by sonsuzus** [comment]
Bu hafta yoklama için bilisimbilsem gmail adresine yazacağınız mailde konu bölümüne adınızı soyadınızı yazıp, içeriğe de "Buradayım" yazın.

**Comment by THEfurkanyildirim** [comment]
İlk başlayan kişi ortadaki taşı alırsa oyunun geri kalanında taş dizilimi 6+6 lık bloklar şeklinde
(...... ......) olur ve 2. oyuncunun yaptığı her hamleyi karşısındaki bloğa kopyalarsa son taş ona kalır
